WordPressKit/PostServiceRemoteREST.m

```diff
@@ -564,4 +589,42 @@ - (NSArray *)tagNamesFromJSONDictionary:(NSDictionary *)jsonTags {
     return [jsonTags allKeys];
 }
 
+/**
+ *  @brief  Returns an array of RemoteUser based on provided JSON
+ *          representation of users.
+ *
+ *  @param  jsonUsers   An array containing JSON representations of users.
+ */
+- (NSArray<RemoteUser *> *)remoteUsersFromJSONArray:(NSArray *)jsonUsers
+{
+    return [jsonUsers wp_map:^id(NSDictionary *jsonUser) {
+        return [self remoteUserFromJSONDictionary:jsonUser];
+    }];
+}
+
+/**
+ *  @brief      Creates a RemoteUser instance based on provided JSON object.
+ *
+ *  @discussion Expected dictionary contents (and its mapping to the
+ *              RemoteUser object):
+ *              - ID -> userID
+ *              - login -> username
+ *              - name -> displayName
+ *              - site_ID -> primaryBlogID
+ *              - avatar_URL -> avatarURL
+ *
+ *  @param  jsonUser    The dictionary that represents a RemoteUser.
+ */
+- (RemoteUser *)remoteUserFromJSONDictionary:(NSDictionary *)jsonUser
```

Comment:
From your success test, it looks like `email` is included in the response. Not sure if it is ever actually provided, but just in case, can we set `user.email` if it's present and default to "" if not?

Reply:
Ah yes, I initially thought that too, but the problem is the `email` parameter returned from the API is a boolean. Here's an example of a complete user JSON from post likes endpoint:
```json
{
    "found": 1,
    "i_like": true,
    "site_ID": 0,
    "post_ID": 1,
    "likes": [
        {
            "ID": 106059135,
            "login": "dvdchr",
            "email": false,
            "name": "dvdchr",
            "first_name": "David",
            "last_name": "Christiandy",
            "nice_name": "dvdchr",
            "URL": "",
            "avatar_URL": "https://0.gravatar.com/avatar/9d5db177ff5c19b2c4806093ec6d87f7?s=96&d=identicon&r=G",
            "profile_URL": "https://en.gravatar.com/dvdchr",
            "ip_address": false,
            "site_ID": 174074163,
            "site_visible": true,
            "default_avatar": false
        }
    ]
}

```
In fact, I don't know what the boolean value in the `email` field represents, as I find that all of the JSON objects have `false` value for every entry. 😅

Reply:
Ah, I see. Yea, that's a little odd. 🤷 OK. We'll leave it for now. I'm not sure if we'll ever need it in this scenario, but if so we can come back to it. Thanks for checking!